R/utils.R

```diff
@@ -291,6 +291,20 @@ check_sampler_transitions_treedepth <- function(post_warmup_sampler_diagnostics,
   }
 }
 
+check_bfmi <- function(post_warmup_sampler_diagnostics) {
+  if (!is.null(post_warmup_sampler_diagnostics)) {
+    energy <- posterior::extract_variable_matrix(post_warmup_sampler_diagnostics, "energy__")
+    ebfmi <- apply(energy, 2, function(x) {
+      (sum(diff(x)^2)/length(x))/var(x)
```

Comment:
```suggestion
      (sum(diff(x)^2)/length(x))/stats::var(x)
```

That will fix the R cmd check NOTE here: 
https://github.com/stan-dev/cmdstanr/pull/500/checks?check_run_id=2589978139#step:16:63